src/connectedmachine/azext_connectedmachine/tests/latest/test_RunCommand_scenario.py

```diff
@@ -0,0 +1,104 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+
+import os, json
+from azure.cli.testsdk import ScenarioTest
+from azure.cli.testsdk import ResourceGroupPreparer
+from .example_steps import step_private_link_resource_list
+from .example_steps import step_private_link_scope_create
+from .example_steps import step_private_link_scope_update
+from .example_steps import step_private_link_scope_show
+from .example_steps import step_private_link_scope_list
+from .example_steps import step_private_link_scope_list2
+from .example_steps import step_private_link_scope_update_tag
+from .example_steps import step_private_endpoint_connection_update
+from .example_steps import step_private_endpoint_connection_list
+from .example_steps import step_private_endpoint_connection_show
+from .example_steps import step_private_endpoint_connection_delete
+from .example_steps import step_private_link_scope_delete
+from .. import (
+    try_manual,
+    raise_if,
+    calc_coverage
+)
+from azure.cli.testsdk.scenario_tests import (
+    live_only
+)
+
+TEST_DIR = os.path.abspath(os.path.join(os.path.abspath(__file__), '..'))
+
+
+class RunCommandScenarioTest(ScenarioTest):
+    @live_only()
+    @ResourceGroupPreparer(name_prefix='cli_test_runcommand')
```

Comment:
Because some commands need to have subscription as input and are only valid for certain subscriptions. If I don't mark it as @live_only(), it fails on CI.